Release/samples/DSC/dsc_rest_client/dsc_rest_client.cpp

```diff
@@ -0,0 +1,102 @@
+//#define BOOST_ASIO_SAMPLE
+
```

Comment:
You can ignore the 2 samples - I will remove them once I start adding actual tests. Right now they are just a quick way for me to experiment with the code.